### PR TITLE
Add request analytics table for analysis of web requests.

### PR DESCRIPTION
**Pull request by ryanhugh**
Note: This code is unlikely to stay in the codebase for a while, just want to do some analysis with kibana and grafana, both which can pull data from ES

### CODE REVIEW

File: backend/elastic.js [hunk 1]
```diff
@@ -43,6 +44,35 @@ class Elastic {
...
+      });
+    }
+  }
+
+  /**
+   * Inserts a singular row into an index.
+   * @param  {string} indexName The index to insert into
+   * @param  {Object} row   The new row to insert into the index.
+   */
+  index(indexName, row) {
```
Comment (gamburgm) [inline backend/elastic.js]: Think you could change the naming conventions here? I got a little confused with the terminology and with `row`.  Maybe naming the method something like `insertDoc` and changing `row` to `doc`?

File: backend/server.js [hunk 2]
```diff
@@ -91,6 +95,47 @@ app.use((req, res, next) => {
...
+    'sec-fetch-site',
+    'sec-fetch-mode',
+    'sec-fetch-user',
+    'accept-encoding',
+    'accept-language',
+  ];
+
+  for (const header of Object.keys(objectToLog)) {
+    // Don't log these headers because they are not useful for analysis.
+    if (removeHeaderList.includes(header)) {
```
Comment (gamburgm) [inline backend/server.js]: Might be cleaner to use `_.omit`

**Review by gamburgm — COMMENTED** [review]
Looks great, just had a few small comments :)

File: backend/requestMapping.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      "sessionToken": {
+        "type": "keyword"
+      },
+      "windowInnerWidth": {
+        "type": "integer"
+      },
+      "windowInnerHeight": {
+        "type": "integer"
+      },
+      "windowScrollY": {
```
Comment (gamburgm) [inline backend/requestMapping.json:40]: But where's `windowScrollX`????????? >:|

Comment (ryanhugh) [reply backend/requestMapping.json:40]: lololol, i'd add it just for you, but i don't think you can even scroll sideways ?

**Review by gamburgm — APPROVED** [review]

**Pull request merged**